beanstalk_worker/worker_start_scan.py

```diff
@@ -101,6 +102,35 @@ def run_a_scanner(self, scanner, image_under_test):
         # if scanner failed to run, this will return {}, do check at receiver end
         return json_data
 
+    def export_scanner_logs(self, scanner, data):
+        """
+        Export scanner logs in given directory
+        """
+        logs_file_path = os.path.join(
+                self.job_info["logs_dir"],
+                constants.SCANNERS_RESULTFILE[scanner][0])
+        logger.log(
+                level=logging.INFO,
+                msg="Scanner=%s result log file:%s" % (scanner, logs_file_path)
+                )
+        try:
+            fin = open(logs_file_path, "w")
+        except IOError as e:
+            logger.log(
+                level=logging.CRITICAL,
+                msg="Failed to write scanner=%s logs on NFS share." % scanner)
```

Comment:
I don't understand why are we categorizing the two errors, if (at the first place) a file can not be opened for writing, we failed writing to file, opening a file in write mode and failing to do so, is *fail to write*. IOError encapsulate that.